config/webpack/app-config.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
     backbone: 'window.Backbone',
     jquery: 'window.$',
     lodash: 'window._',
+    'champaign-i18n': 'window.I18n',
```

Comment:
This is where we map to the global, so `import { translations } from 'champaign-i18n';` is the equivalent to `const translations = window.I18n.translations`;

In non-rails testing environments where we don't actually have the window.I18n object (i.e. during Jest tests), we map that module to a stub that returns the actual yaml files in our repo (so, without the external assets, which we don't need during tests). See package.json:101 for that: `"^champaign-i18n$": "<rootDir>/spec/jest/champaign-i18n-stub.js"`